tests/trading/test_wcl_fuzzy.py

```diff
@@ -296,58 +317,58 @@ def execute_askOrder_tests(contractsFixture, universe, market, fxpAmount, fxpPri
         expectedFillerShortShares = 0,
         expectedFillerTokens = 0)
 
-def test_binary(contractsFixture, market, randomAmount, randomNormalizedPrice):
+    fixture.resetToSnapshot(kitchenSinkSnapshot)
```

Comment:
No need to reset the snapshot at the end, just at the beginning (see note above about moving the resets into the `execute` function).